source/index.md

```diff
@@ -600,9 +602,41 @@ object][ApplePay-PaymentToken] for more information.
   subsequent recurring authorization.
 </p>
 
-##### Method: `mobilepayonline`
 
-<dl class="dl-horizontal">
+---
+##### **Method**: `googlepay`
+
+To accept a payment using Google Pay, apart from the complete payment method
+token and merchant ID, the derived shared secret is required.
+Please refer to the [official documentation][GooglePay-PaymentCryptography].
+
+<dl>
+  <dt>googlepay[token]</dt>
+  <dd>[:json:]<br />Raw payment method token as received in response from Google. UTF-8
+    encoded serialization of a JSON dictionary.</dd>
+  <dt>googlepay[shared_secret]</dt>
+  <dd>[:base64:]<br />The shared secret derived from the ephemeral public key and
+    your private key.</dd>
+  <dt>googlepay[recipient_id]</dt>
+  <dd>merchant:[:merchant_id:]<br />The ID of the merchant preprended with "merchant:".</dd>
```

Comment:
Oh, wait. Why don't *we* just append that `"merchant:"` string and require the "actual" merchant ID to be given in the API?

Reply:
Yeah. You're absolutely correct. I can't believe I didn't consider this.. :keyboard: :timer_clock: 
Working on it now.

Reply:
:heavy_check_mark: